File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-10.java
class Solution {
    public int findMin(int[] nums) {
        int n=nums.length;

        int l=0,h=n-1;
        int m=0;
        while(l<h)
        {
            m=(l+h)/2;

           if(nums[m]<nums[h])
           {
            h=m;
           }
           else
            l=m+1;

        }
        return nums[l];

    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-10.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        

        index = -1
        for i in range(0,len(nums)):
            rem = target - nums[i]

            if rem in nums :
                try:
                    index = nums.index(rem,i+1)
                except Exception as e:
                    print(i," ",rem)
                    continue
                return [i,index]
        